### PR TITLE
Dev frontend

### LINKED ISSUE #10: Connect Movie DB API
https://developers.themoviedb.org/3/getting-started/introduction

### PR DESCRIPTION
**Pull request by Hunajpu**
Frontend changes from issue #2 and #10

### CODE REVIEW

File: app/static/styles/main.css [hunk 1]
```diff
@@ -1,24 +1,71 @@
...
+  font-size: 20px;
+  margin: 10px;
+  color: white;
+}
+
 /* ========================= friendlist ========================= */
 
 .card-user-card {
-    border-radius: 50%;
+  border-radius: 50%;
```
Comment (truongnguyenlinh) [inline app/static/styles/main.css:51]: Seems like we all have different formatting preferences. @nanonite9 do you prefer a 4 space indentation or 2? We can decide collectively as a team.

File: app/__init__.py [hunk 2]
```diff
@@ -53,19 +103,82 @@ def home():
...
+        "flixlist.html",
+        title="FlixList",
+        menu_item=menu_item,
+        movies_data=movies_data,
+        info_to_display=info_to_display,
+    )
+
 
 @app.route("/friendlist")
 def friendflix():
```
Comment (ajenaration) [inline app/__init__.py]: I changed the function name to **friendlist**. A conflict may arise.

File: app/__init__.py [hunk 3]
```diff
@@ -53,19 +103,82 @@ def home():
...
     rows = 3
-    return render_template("friendflix.html", title="Friend List", rows=rows)
+    return render_template("friendlist.html", title="Friend List", rows=rows)
+
 
 @app.route("/recommendations")
 def recsflix():
```
Comment (ajenaration) [inline app/__init__.py]: I changed the function name to **recommendations**. A conflict may arise.

**Review by ajenaration — COMMENTED** [review]
I changed the function names to **friendlist** and **recommendations**. Conflicts may arise.

File: app/__init__.py (hunk 2 above)
Comment (Hunajpu) [reply app/__init__.py]: Got it, we should keep your function name changes when merging

File: app/__init__.py (hunk 3 above)
Comment (Hunajpu) [reply app/__init__.py]: Got it, we should keep your function name changes when merging

File: app/static/styles/main.css (hunk 1 above)
Comment (Hunajpu) [reply app/static/styles/main.css:51]: I think 4 spaces looks cleaner

Comment (ajenaration) [reply app/static/styles/main.css:51]: I am okay with any. My editor automatically indented to 4.

**Review by ajenaration — APPROVED** [review]

**Review by truongnguyenlinh — APPROVED** [review]
LGTM @Hunajpu please merge main into your branch and squash your commit before merging

**Pull request merged**